### ISSUE TITLE
Resurrect MASKA210R and MASKA430R as allowed NIRCam subarray names

### ISSUE THREAD

**Issue by hbushouse** [STScI-JWST/jwst#1822]
CRDS ticket https://jira.stsci.edu/browse/CRDS-182 indicates that a few old NIRCam subarray names are being resurrected to allow for backwards compatibility with old rmaps and reference files. The 4 values being added back to the allowed list are MASKA210R, MASKA430R, MASKBSWB, and MASKBLWB. The last 2 were already added back into the datamodels schema in #1410. So we now need to readd MASKA210R and MASKA430R.

Also, there's a new NIRSpec subarray name "STRIPE" that should be added to the list of allowed values.

**Comment by hbushouse** [comment]
Daryl Swade also reports:
```
NIS_EXTCAL has been added to the possible values of EXP_TYPE is the SOC PRD SOCSharedFiles/ObservatoryStatusFile.xsd file.
```
So the enum list for EXP_TYPE should also be updated to include NIS_EXTCAL.

**Comment by stscieisenhamer** [comment]
New EXP_TYPE? Please file a new issue to update #1188.

**Comment by hbushouse** [comment]
Fixed in #1851
